File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/Callback/CallbackPostsLists.java
package com.jackson_siro.visongbook.models.Callback;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import com.jackson_siro.visongbook.models.PostModel;

public class CallbackPostsLists implements Serializable{
    public int total = -1;
    public List<PostModel> data = new ArrayList<PostModel>();
}

File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/callback/CallbackPostsLists.kt
package com.jackson_siro.visongbook.models.callback

import java.util.ArrayList
import com.jackson_siro.visongbook.models.PostModel
import java.io.Serializable

class CallbackPostsLists: Serializable{
    lateinit var data: MutableList<PostModel>
    var total: Int? = -1
    //var data: List<PostModel>? = ArrayList()
}
